cluster/charts/fuel-streams-publisher/values.yaml

```diff
@@ -0,0 +1,141 @@
+# Default values for fuel-streams-publisher
```

Comment:
I made few improvements on this helm chart also, deleting stuff we were not using and creating some default values and new template configuration to be easier to configure volumes, etc